source/Calamari.Common/Plumbing/FileSystem/CalamariPhysicalFileSystem.cs

```diff
@@ -206,17 +206,17 @@ public virtual IEnumerable<string> EnumerateFiles(string parentDirectoryPath, pa
             var parentDirectoryInfo = new DirectoryInfo(parentDirectoryPath);
 
             return searchPatterns.Length == 0
-                ? parentDirectoryInfo.GetFiles("*", SearchOption.TopDirectoryOnly).Select(fi => fi.FullName)
-                : searchPatterns.SelectMany(pattern => parentDirectoryInfo.GetFiles(pattern, SearchOption.TopDirectoryOnly).Select(fi => fi.FullName));
+                ? parentDirectoryInfo.EnumerateFiles("*", SearchOption.TopDirectoryOnly).Select(fi => fi.FullName)
+                : searchPatterns.SelectMany(pattern => parentDirectoryInfo.EnumerateFiles(pattern, SearchOption.TopDirectoryOnly).Select(fi => fi.FullName));
```

Comment:
Given that we're only after the full name of the files, we should be able to get a significant performance increase (~2x) if we use the static version of this call, i.e. `Directory.EnumerateFiles(parentDirectoryPath, "*", SearchOption.TopDirectoryOnly)`.
Same applies below too.  At the moment, we're loading all the info about the files, not just their name.

Reply:
Fixed! Turns out this wasn't setup in the correct way (it wasn't using AlphaFS for .net framework builds). I've fixed it so now it's done properly. Once we're off .net framework this `CalamariPhysicalFileSystem` class should be a lot simpler.